app.py

```
@@ -1,13 +1,14 @@
 from flask import Flask, render_template, request, redirect, url_for, session, jsonify
 import sqlite3
 import hashlib
+import time
```

Ruff (0.8.0)

`time` imported but unused

Remove unused import: `time`

(F401)